app/models/reporting/current_workload_report.rb

```diff
@@ -8,23 +8,28 @@ def initialize(work_streams: Types::WorkStreamType.values, business_day_limit: 9
     attr_reader :work_streams
 
     def rows
-      business_days.map do |business_day|
-        CurrentWorkloadReportRow.new business_day: business_day,
-                                     total_received: received_counts.fetch(business_day, 0),
-                                     total_open: open_counts.fetch(business_day, 0)
+      @rows ||= business_days.map do |business_day|
+        CurrentWorkloadReportRow.new(
+          business_day: business_day,
+          total_received: received_counts.fetch(business_day.date, 0),
+          total_open: open_counts.fetch(business_day.date, 0)
+        )
       end
     end
 
     private
 
     def business_days
-      Array.new(@business_day_limit + 1) do |age_in_business_days|
-        BusinessDay.new(age_in_business_days:).date
-      end
+      @business_days ||= BusinessDay.list_backwards(
+        age_limit: @business_day_limit
+      )
     end
 
     def scope
-      Review.where(work_stream: work_streams, business_day: (business_days.last..business_days.first))
+      Review.where(
```

Comment:
Tiny thing could we extract these to variables in the method:
```ruby
first_day = business_days.last.date
last_day = business_days.first.date
```

its easier to parse (`first_day..last_day`)